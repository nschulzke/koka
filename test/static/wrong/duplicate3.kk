// duplicate constructor
reference type dup
  Dup1(:int)
  Dup1(:int)
