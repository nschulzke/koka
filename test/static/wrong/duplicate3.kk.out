test/static/wrong/duplicate3@kk(4, 3): kind error: Constructor static/wrong/duplicate3/Dup1 is already defined at (3, 3)